### PR TITLE
[MPOM-322] Add ASF Data Privacy

### PR DESCRIPTION
**Pull request by slawekjaranowski**
- add link to Data Privacy
- add / copy ASF section to documentation

### CODE REVIEW

File: doxia-tools/src/site-docs/site.xml [hunk 1]
```diff
@@ -37,6 +37,15 @@ under the License.
       <item name="Doxia Tools" href="../doxia-tools/index.html"/>
     </menu>
 
-    <menu ref="parent" />
```
Comment (michael-o) [inline doxia-tools/src/site-docs/site.xml:40]: Can you explain why this is not necessary anymore?

Comment (slawekjaranowski) [reply doxia-tools/src/site-docs/site.xml:40]: few lines above we have list of all modules, so link to parent in this case is duplicated.

Comment (michael-o) [reply doxia-tools/src/site-docs/site.xml:40]: Alright, thanks!

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #264

**Pull request merged**